pegasus/forms/hoc_signup_2016.rb

```diff
@@ -0,0 +1,7 @@
+require pegasus_dir 'forms/hoc_signup_2014'
```

Comment:
That's alright.  It just sounds like this is really forms/hoc_signup at this point, but we're using the original year in the filename which isn't entirely intuitive.  If it's risky or expensive to change, then alright to leave as-is.